module_16/tests/demoblaze_test.robot

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Library  SeleniumLibrary
+Resource  ../resources/resources.robot
+Test Setup  Open Browser    ${url}    ${browser}
+
+*** Test Cases ***
+LoginTest
+    ClickLogInButton
+    ValidateLoginAndPasswordFieldsArePresented
+    LoginToApplication
+    CheckLogOutButtonIsVisible
+    ValidateWelcomeMessageText  Welcome ${login}
+    close browser
+
+AddToCartTest
+    LoginPreconditions
```

Comment:
login part is a preconfition for this scenario so it would be better move it to [Test Setup] section of this test case

Reply:
Thanks! Fixed:
![image](https://github.com/sergei820/PythonTAProgramEpam/assets/56273100/e5cc8b5b-3c82-4e03-833d-a26b98d2bdb2)